python/sgl_jax/srt/model_executor/model_runner.py

```diff
@@ -148,7 +148,7 @@ def initialize(self):
             self.is_hybrid = True
 
         # Init lora
-        if server_args.enable_lora:
+        if server_args.enable_lora or server_args.enable_single_lora:
             self.init_lora_manager()
```

Comment:
When use enable_single_lora, set `enable_lora` = True, and assert `max_loras_per_batch` to 1. I think it's better way.